dace/codegen/targets/rtl.py

```diff
@@ -154,14 +154,13 @@ def unparse_rtl_tasklet(sdfg, state_id, dfg, node, function_stream,
             idx_str = ""
             # catch symbolic (compile time variables)
             if symbolic.issymbolic(tasklet.in_connectors[inp].veclen, sdfg.constants):
-                pass
-                # TODO: raise RuntimeError("Please use sdfg.specialize to specialize the symbol in expression: {}".format(tasklet.in_connectors[inp].veclen))
+                raise RuntimeError("Please use sdfg.specialize to specialize the symbol in expression: {}".format(tasklet.in_connectors[inp].veclen))
             if symbolic.issymbolic(tasklet.in_connectors[inp].bytes, sdfg.constants):
-                pass
-                # TODO: raise RuntimeError("Please use sdfg.specialize to specialize the symbol in expression: {}".format(tasklet.in_connectors[inp].bytes))
+                raise RuntimeError("Please use sdfg.specialize to specialize the symbol in expression: {}".format(tasklet.in_connectors[inp].bytes))
```

Comment:
I would make a function that takes `expr.free_symbols` and subtracts from it the keys in `sdfg.constants`, in order to print out the symbols that need to be specialized.

Reply:
Good idea. I will do that.